### PR TITLE
feat(curriculum): add Introductions in team meeting block 1.2

### PR DESCRIPTION
**Pull request by nieldakarla**
Challenges for the second block of the English Curriculum. 

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Ref: #52400

### CODE REVIEW

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657b400653813a1caa228aca.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+# --question--
+
+## --text--
+
+What does the expression `I’m happy` indicate in the present simple tense?
+
+## --answers--
+
+`A future plan`
```
Comment (moT01) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657b400653813a1caa228aca.md:27]:
```suggestion
A future plan
```

**Review by moT01 — COMMENTED** [review]
This is looking good @nieldakarla 🎉 

I just wrote down the things I found instead of making code suggestion in this UI. Here they are:

Task 3:
Answers should not be in backticks

Task 19:
Answers 3 and 4 are mixed up.

Task 40:
`User` should not be capitalized in the fill in the blank sentence

Task 44:
We shouldn’t use `+`’s in the description. This could be an alternative:

```
Phrases like `I really like [subject]`, with `subject` being a subject you like, or `I can't wait to do [activity]`, with `activity` being an activity you want to do, are sentences that show your excitement and interest about something.
```

Task 51:
`let` in the first blank answer should be capitalized

Task 52:
`this` in the first blank answer should be capitalized

Task 71:
`OK` -> `Okay`? - probably fine to leave it.

Task 81:
Missing period at end of fill in the blank sentence.

Task 86:
I think the answer is supposed to be 2 instead of 1

Task 88:
Has a dash in the challenge title: `Task-88` -> `Task 88`. Same issue with the title in the `meta.json` file.

Task 98:
In the blank answer `point`, We should remove the space in the answer.

**Comment by nieldakarla** [comment]
I'll implement the changes now, @moT01. And, thank you for the alternative description you gave me. It was very helpful!

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657cd9ae412af80b2a7ff265.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+<!--
+AUDIO REFERENCE:
+You can talk to her any time, even in this virtual setting, if you have any questions, Lisa.
+-->
+
+# --description--
+
+`Any` is used to indicate no matter which one, no matter when, or no matter what. For example, `You can ask me any question` means you can ask all kinds of questions.
+
+`Even` is used to emphasize something unexpected or to include something in addition. For example, `She can speak many languages, even Tupí` means that it's surprising or notable that she can also speak Tupi, in addition to other languages.
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657cd9ae412af80b2a7ff265.md:17]: No stress mark on Tupi within the backticks

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657ce691d6c57c107e650c5e.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+Bob is introducing Tom as the new graphic designer.
+
+---
+
+Bob is asking Tom to lead the next team meeting.
+
+### --feedback--
+
+THis line is about Tom's role and expertise, not about leading meetings.
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657ce691d6c57c107e650c5e.md:41]: lowercase the "H" in "This"

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657daa5ab8505427a5b99cd2.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+# --question--
+
+## --text--
+
+What is Bob's role in the company?
+
+## --answers--
+
+He is the team’s project manager.
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657daa5ab8505427a5b99cd2.md:25]: Watch for the curly quote in "team's"

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657daa5ab8505427a5b99cd2.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+---
+
+He is the team’s graphic designer.
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657daa5ab8505427a5b99cd2.md:29]: Here too

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657daa5ab8505427a5b99cd2.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+He is the new intern.
+
+### --feedback--
+
+Bob introduces himself with a specific management role, not as an intern.
+
+---
+
+He is the company’s CEO.
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657daa5ab8505427a5b99cd2.md:45]: And here

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657dadfc7d21eb294c9f057e.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+`I'm the person you always need to talk to`
+
+### --feedback--
+
+This part introduces Bob as the point of contact but doesn't specify what for.
+
+---
+
+`And I’m Bob, the team’s project manager`
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657dadfc7d21eb294c9f057e.md:33]: Curly quotes

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657daf702ed04a29ee42de69.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+# --description--
+
+Listen to the audio and fill in the missing words.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`Hi, Bob. It’s _ to meet you _.`
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657daf702ed04a29ee42de69.md:21]: Curly quotes

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657df21cc45b1f66112fb8fc.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+
+# --description--
+
+Listen to the audio and fill in the missing words.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`_! Lisa, it’s great to _ you _ our team, _ if just online.`
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657df21cc45b1f66112fb8fc.md:21]: Curly quotes

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657df5d14291b56887825276.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+
+# --description--
+
+Listen to the audio and fill in the missing words.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`I’m James and I’m _ to _ you with your _.`
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657df5d14291b56887825276.md:22]: Curly quotes

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657df6018a70e468f5dc016a.md [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+
+What is James communicating?
+
+## --answers--
+
+That he is the team leader.
+
+### --feedback--
+
+James doesn't metion being a team leader.
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657df6018a70e468f5dc016a.md:29]: "mention" is misspelled

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e3ca1c4d9636e9012e1a9.md [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+
+To discuss her work experience in depth.
+
+### --feedback--
+
+He is not asking for a detailed discussion of work experience.
+
+---
+
+To provide a detailed report of her team’s performance.
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e3ca1c4d9636e9012e1a9.md:41]: Curly quotes

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e4b7d381d567e8d97967c.md [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+
+`Efforts` are the attempts or hard work put into doing something. Like, `His efforts in the project were noticeable`. 
+
+`Coordinating efforts` in a project means organizing everyone's work so the project is successful. It's like making sure all parts of a puzzle fit together.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`I'm responsible for the _ aspects of the project. He’s responsible for _ our _ to meet your needs.`
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e4b7d381d567e8d97967c.md:25]: Curly quotes in "he's"

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e4db2e3fc8d7fb41b8b85.md [hunk 14]
```diff
@@ -0,0 +1,29 @@
...
+
+# --description--
+
+Listen to the audio and fill in the missing word.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`Hey, Bob. How’s it _?`
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e4db2e3fc8d7fb41b8b85.md:21]: Curly quotes in "how's"

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e4fd2ecf31280ef673f0d.md [hunk 15]
```diff
@@ -0,0 +1,53 @@
...
+
+# --question--
+
+## --text--
+
+What does Linda do?
+
+## --answers--
+
+She manages the company’s finances.
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e4fd2ecf31280ef673f0d.md:25]: Curly quotes

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e51218ff18f8191b76ea9.md [hunk 16]
```diff
@@ -0,0 +1,57 @@
...
+
+`Wonderful` is an adjective used to express that something is very good or pleasant. 
+
+`Corp.` is an abbreviation for `corporation`, which is a large company or group of companies.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`_ to _ you all. I'm David, the project _ from FCC _`
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e51218ff18f8191b76ea9.md:25]: Missing a full stop at the end of the sentence.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e5477269b94834908826f.md [hunk 17]
```diff
@@ -0,0 +1,39 @@
...
+
+# --description--
+
+`Collaboration` refers to working together with others to achieve a common goal.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`I’ll be your _ of contact and I’ll help make sure our _ is great.`
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e5477269b94834908826f.md:23]: Curly quotes in both "I'll"

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e56e7034d2d858b6e9e00.md [hunk 18]
```diff
@@ -0,0 +1,57 @@
...
+
+In David's statement, what do the two uses of `we` refer to? 
+
+## --answers--
+
+Both refer to David and his team at FCC Corp. 
+
+### --feedback--
+
+The second `we` is not just about FCC corp.
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e56e7034d2d858b6e9e00.md:33]: Uppercase the C in corp

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e59605c6f688785fbb46d.md [hunk 19]
```diff
@@ -0,0 +1,55 @@
...
+
+She is proposing to start discussing the project details and David's needs for the project.
+
+---
+
+She is asking David to take over the meeting.
+
+### --feedback--
+
+Maria' is not asking David to take over the meeting.
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e59605c6f688785fbb46d.md:43]: Remove this ' after Maria

**Review by DanielRosa74 — CHANGES_REQUESTED** [review]
Minor tweaks needed. LGTM overall

**Review by DanielRosa74 — APPROVED** [review]
LGTM

**Comment by moT01** [comment]
I pushed the updated titles to the meta.json file

**Review by moT01 — APPROVED** [review]

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e4e3b02a2128049c344c8.md [hunk 20]
```diff
@@ -0,0 +1,37 @@
+---
+id: 657e4e3b02a2128049c344c8
+title: Task 93
+challengeType: 22
+dashedName: task-93
+audioPath: Acurriculum/js-music-player/We-Are-Going-to-Make-it.mp3
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e4e3b02a2128049c344c8.md:6]:
```suggestion
audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e523fcaaa0e823b3789e5.md [hunk 21]
```diff
@@ -0,0 +1,55 @@
...
+
+He is a project consultant at FCC Corp.
+
+### --feedback--
+
+While David is involved in projects, this is not his role.
+
+---
+
+He is doctor from FCC Corp.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657e523fcaaa0e823b3789e5.md:43]:
```suggestion
He is a doctor from FCC Corp.
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Couple of nits, then happy to merge.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657ca36eacc797817451ece6.md [hunk 22]
```diff
@@ -0,0 +1,54 @@
...
+id: 657ca36eacc797817451ece6
+title: Task 27
+challengeType: 19
+dashedName: task-27
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE:
+Bob: Can you hand me that file, please? I need to review it before our meeting.
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657ca36eacc797817451ece6.md:11]: This has passed me by in the first review. This line is not in the dialogues, so there's no audio for that.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657cd9ae412af80b2a7ff265.md [hunk 23]
```diff
@@ -0,0 +1,49 @@
...
+You can talk to her any time, even in this virtual setting, if you have any questions, Lisa.
+-->
+
+# --description--
+
+`Any` is used to indicate no matter which one, no matter when, or no matter what. For example, `You can ask me any question` means you can ask all kinds of questions.
+
+`Even` is used to emphasize something unexpected or to include something in addition. For example, `She can speak many languages, even Tupi` means that it's surprising or notable that she can also speak Tupi, in addition to other languages. 
+
+The word even highlights something extra or beyond what is usual.
```
[the remark this answers is not in the document]
Comment (naomi-lgbt) [reply curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657cd9ae412af80b2a7ff265.md:19]: Yes

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**